src/pydata_sphinx_theme/assets/styles/components/_prev-next.scss

```diff
@@ -30,15 +30,10 @@
       font-size: 1.1em;
     }
 
-    &:hover p.prev-next-title {
-      @include link-style-hover;
-    }
-    // Exception here - keep visited in the default link colour
-    // it still should hover on the default link hover colour
-    &:visited p.prev-next-title {
-      color: var(--pst-color-link);
-      &:hover {
-        color: var(--pst-color-link-hover);
```

Comment:
This block of style rule exceptions isn't needed because:

- line 41 is incorporated in lines 34 and 36 on the right - the `link-style-hover` mixin (modified and shown above in this diff) sets color to `--pst-color-link-hover`
- line 39 is incorporated in lines 27 and 28 on the right - the [link-style-default mixin sets color to --pst-color-link](https://github.com/pydata/pydata-sphinx-theme/blob/4087f896c0dcd351114309d8d426cf34cdd280f2/src/pydata_sphinx_theme/assets/styles/abstracts/_links.scss#L69)